sihl/src/contract_user.ml

```diff
@@ -52,8 +71,28 @@ module type Sig = sig
       :  ?cookie_key:string
       -> ?secret:string
       -> ?key:string
-      -> Rock.Request.t
+      -> Opium.Request.t
       -> t option Lwt.t
+
+    (** [user_to_session ?cookie_key ?secret ?key user response] returns a
+        response with a [user] associated to the session. Any previously set
+        session is overwritten. This is typically part of the login workflow.
+
+        [cookie_key] is the name/key of the session cookie. By default, the
+        value is [_session].
```

Comment:
I don't think you are setting that default in the function implementation

Reply:
It is by `web_session`, we are forwarding the optional.